server/src/test/java/io/spine/server/commandbus/AbstractCommandBusTestSuite.java

```diff
@@ -72,10 +72,8 @@
 
 /**
  * Abstract base for test suites of {@code CommandBus}.
- *
- * @author Alexander Yevsyukov
  */
-@SuppressWarnings("ProtectedField") // OK for brevity of derived tests.
+@SuppressWarnings("ProtectedField") // for brevity of derived tests.
```

Comment:
Please capitalize the comment.

Reply:
The comment here is continuation of the sentence, not the start.